input.go

```diff
@@ -6,15 +6,15 @@ import "github.com/relvacode/iso8601"
 
 // AlertSourceExternalIdentifier specifies the input needed to find an alert source with external information.
 type AlertSourceExternalIdentifier struct {
-	Type       AlertSourceTypeEnum `json:"type" yaml:"type" example:"pagerduty"`                                   // The type of the alert. (Required.)
-	ExternalId string              `json:"externalId" yaml:"externalId" example:"Z2lkOi8vc2VydmljZS8xMjM0NTY3ODk"` // The external id of the alert. (Required.)
+	ExternalId string              `json:"externalId" yaml:"externalId" example:"example_value"` // The external id of the alert. (Required.)
```

Comment:
why is the example value in the struct tag changing?

Reply:
This is an edge case that needs special handling. The `Z2lk...` value is populated for fields of type `Id`. In the API schema this is string but I believe we expect it to have a value like an Id under the hood.